CONTRIBUTING.md

```diff
@@ -277,6 +277,24 @@ COVERAGE_PROCESS_START DEFAULT=/coverage.rc
 
 The CI/CD system is based on [Drone platform](https://drone.io/) and the server is  https://ci.vmware.run/. More information is found at our [CI.md](https://github.com/vmware/docker-volume-vsphere/blob/master/CI.md)
 
+## Release naming convention
+The following is the plan for releases for GA and post GA. Before GA, the release name format remains the same.
+
+### Type of Release:
+We plan to have two types of releases: edge release and stable release.
+
+Edge release: A monthly release with new features and bug fixes.
+
```

Comment:
nit: no need for extra line

Reply:
That line is needed, otherwise, the line "edge release" and "stable release" will be in the same line when browsing the md file.

Reply:
got it. Thanks for checking!